app/models/statistics_report.rb

```diff
@@ -0,0 +1,30 @@
+class StatisticsReport
+  attr_reader :start_date
+  attr_reader :end_date
+
+  REPORTS_KLASSES = [StatisticsReport::AuctionData,
```

Comment:
`REPORT_CLASSES`?

Reply:
That's intended as a habit not to use keyword 'class' and use 'klass' instead.